paimon-core/src/main/java/org/apache/paimon/AbstractFileStore.java

```diff
@@ -53,13 +53,17 @@
 import java.util.Comparator;
 import java.util.List;
 
+import static org.apache.paimon.utils.BranchManager.DEFAULT_MAIN_BRANCH;
+
 /**
  * Base {@link FileStore} implementation.
  *
  * @param <T> type of record to read and write.
  */
 public abstract class AbstractFileStore<T> implements FileStore<T> {
 
+    public static final String DEFAULT_MAIN_BRANCH = "main";
```

Comment:
Why not use `BranchManager.DEFAULT_MAIN_BRANCH` directly here.

Reply:
I have fixed it. Thanks. @xiangyuf